static/docs/commands-reference/repro.md

```diff
@@ -57,6 +57,11 @@ specified), and updates stage files with the new checksum information.
   reproduced as part of the pipeline in the parent directory, or as an
   independent unit.
 
+- `-R`, `--recursive` - `targets` is expected to contain directory path(s).
```

Comment:
it actually accepts files as well now, just to simplify and make the workflow flexible

Reply:
Hmm but if `targets` only includes files and `-R` is used, it's basically ignored, right? I'll update the paragraph for your consideration assuming so for now...

Reply:
right, it will be just ignored.

Reply:
OK paragraph changed (for all cmd refs) please check it out and lmk if you think it needs improvement in a new comment. (Resolving this one since it will be marked as outdated by Github.)